blockstack_client/profile.py

```diff
@@ -195,90 +176,112 @@ def load_name_profile(name, user_zonefile, data_address, owner_address, use_zone
     """
     # get user's data public key
     try:
-        user_data_pubkey = user_db.user_zonefile_data_pubkey( user_zonefile )
-    except ValueError, v:
+        user_data_pubkey = user_db.user_zonefile_data_pubkey(user_zonefile)
+    except ValueError as v:
         # user decided to put multiple keys under the same name into the zonefile.
         # so don't use them.
         log.exception(v)
-        user_data_pubkey = None 
+        user_data_pubkey = None
 
     if user_data_pubkey is None and data_address is None and owner_address is None:
-        raise Exception("Missing user data public key and address; cannot verify profile")
+        msg = 'Missing user data public key and address; cannot verify profile'
+        raise Exception(msg)
 
     if user_data_pubkey is None:
-        log.warn("No data public key set; falling back to hash of data and/or owner public key for profile authentication")
+        msg = (
+            'No data public key set; falling back to hash of data '
+            'and/or owner public key for profile authentication'
+        )
+        log.warn(msg)
```

Comment:
For reference. It is better to use a msg variable. This works well if you need to do additional formatting.